From c61f68f8110d29726f99409abc0b10d6e45d2d1c Mon Sep 17 00:00:00 2001
From: Thomas Guillem <thomas@gllm.fr>
Date: Fri, 13 Nov 2015 18:01:45 +0100
Subject: [PATCH 17/20] avcodec: fix orientation overridden to 0

This commit fixes a regression introduced by
adf9ce6dc626ef25ef94b8adb80e764cfbc86a50
---
 modules/codec/avcodec/video.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/codec/avcodec/video.c b/modules/codec/avcodec/video.c
index bd65a5e..c70bccf 100644
--- a/modules/codec/avcodec/video.c
+++ b/modules/codec/avcodec/video.c
@@ -208,6 +208,7 @@ static int lavc_UpdateVideoFormat(decoder_t *dec, AVCodecContext *ctx,
     es_format_Clean(&dec->fmt_out);
     es_format_Init(&dec->fmt_out, VIDEO_ES, fmt_out.i_chroma);
     dec->fmt_out.video = fmt_out;
+    dec->fmt_out.video.orientation = dec->fmt_in.video.orientation;;
     return decoder_UpdateVideoFormat(dec);
 }
 
-- 
2.6.3

